luci-app-olsrd2/Makefile

```diff
@@ -0,0 +1,57 @@
+# call BuildPackage - OpenWrt buildroot signature
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luci-app-olsrd2
+PKG_VERSION:=0.2.6
+PKG_RELEASE:=14
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/luci-app-olsrd2
+  SECTION:=luci
+  CATEGORY:=LuCI
+  SUBMENU:=3. Applications
+  TITLE:=OLSR2 configuration and status module
+  MAINTAINER:=Patrick Grimm <patrick@lunatiki.de>
+  EXTRA_DEPENDS:=oonf-olsrd2, luci-mod-admin-full
+  PKGARCH:=all
+endef
+
+define Build/Prepare
+endef
+
+define Build/Configure
+endef
+
+define Build/Compile
+endef
+
+define Package/$(PKG_NAME)/postinst
```

Comment:
You are reusing ``PKG_NAME`` variable, which was removed by treewide commit in packages repository several years ago. It makes things harder to review.

So you should use here:
define Package/luci-app-olsrd2/postinst